webui/react/src/e2e/models/ant/Pagination.ts

```diff
@@ -0,0 +1,48 @@
+import { Locator } from '@playwright/test';
+
+import { BaseComponent, NamedComponent } from 'e2e/models/BaseComponent';
+import { Select } from 'e2e/models/hew/Select';
+
+/**
+ * Returns the representation of a Table Pagination from Ant.
+ * This constructor represents the Table in antd/es/pagination/index.d.ts.
+ * @param {object} obj
+ * @param {CanBeParent} obj.parent - The parent used to locate this Pagination
+ * @param {string} obj.selector - Used as a selector uesd to locate this object
+ */
+export class Pagination extends NamedComponent {
+  readonly defaultSelector = '.ant-pagination';
+  readonly previous = new BaseComponent({
+    parent: this,
+    selector: 'li.ant-pagination-prev',
+  });
+  readonly next = new BaseComponent({
+    parent: this,
+    selector: 'li.ant-pagination-next',
+  });
+  readonly #options: BaseComponent = new BaseComponent({
```

Comment:
If it has a `#` it should be `protected readonly` right?

Reply:
`#` is for `private`. I'm just using it so it doesn't show up in my autocomplete.